src/Handler/Table/Import/ImportTableFromTableHandler.php

```diff
@@ -0,0 +1,247 @@
+            //    $dedupColumns,
+            //);
+            throw new LogicException('Deduplication is not implemented.');
+        }
+
+        $isFullImport = $options->getImportType() === ImportType::FULL;
+        $insertOnlyDuplicates = $options->getDedupType() === ImportOptions\DedupType::INSERT_DUPLICATES;
+        if ($isFullImport && $insertOnlyDuplicates && !$importOptions->useTimestamp()) {
+            // when full load is performed with no deduplication only copy data using ToStage class
+            // this will skip moving data to stage table
+            // this is used on full load into workspace where data are deduplicated already
+            $toStageImporter = new ToStageImporter($bqClient);
+            $importState = $toStageImporter->importToStagingTable(
+                $source,
+                $destinationDefinition,
+                $importOptions
+            );
+            return [null, $importState->getResult()];
+        }
+
+        $stagingTable = $this->createStateTable($destinationDefinition, $sourceMapping, $bqClient);
+        // load to staging table
+        $toStageImporter = new ToStageImporter($bqClient);
+        $importState = $toStageImporter->importToStagingTable(
+            $source,
+            $stagingTable,
+            $importOptions
+        );
+        // import data to destination
+        $toFinalTableImporter = new FullImporter($bqClient);
+        if ($importOptions->isIncremental()) {
+            throw new LogicException('Not implemented');
+            //$toFinalTableImporter = new IncrementalImporter($db);
+        }
+        $importResult = $toFinalTableImporter->importToTable(
+            $stagingTable,
+            $destinationDefinition,
+            $importOptions,
+            $importState
+        );
+        return [$stagingTable, $importResult];
+    }
+
+    private function createStateTable(
+        BigqueryTableDefinition $destinationDefinition,
+        TableImportFromTableCommand\SourceTableMapping $sourceMapping,
+        BigQueryClient $bqClient
+    ): BigqueryTableDefinition {
+        // prepare staging table definition
+        /** @var TableImportFromTableCommand\SourceTableMapping\ColumnMapping[] $mappings */
+        $mappings = iterator_to_array($sourceMapping->getColumnMappings()->getIterator());
+        $stagingTable = StageTableDefinitionFactory::createStagingTableDefinitionWithMapping(
+            $destinationDefinition,
+            $mappings
+        );
+        // create staging table
+        $qb = new BigqueryTableQueryBuilder();
+        $bqClient->runQuery($bqClient->query(
+            $qb->getCreateTableCommand(
+                $stagingTable->getSchemaName(),
+                $stagingTable->getTableName(),
+                $stagingTable->getColumnsDefinitions(),
+                [] //<-- dont create stage table with primary keys to allow duplicates
+                // @todo maybe set primary keys if source table has them
```

Comment:
Tady to pro BQ neplatí ta PK nemá.
```suggestion
                [] //<-- there are no PK in BQ
```